README.md

```diff
@@ -8,3 +8,6 @@ A repository for machine learning course in sbme department
 [![Download](https://img.shields.io/static/v1?label=notebook&message=download&color=green)](https://Mohamed-Ibrahim-01.github.io/ml-tasks/A1_ML_intro.ipynb)
 [![Preview](https://img.shields.io/static/v1?label=notebook&message=preview&color=orange)](https://github.com/Mohamed-Ibrahim-01/ml-tasks/blob/master/A1_ML_intro.ipynb)
 
+
+this is a change made be AbdelrahmanKK
```

Comment:
Good commit to test the pull request, but it won't be merged as it messes up the readme :)
Try to add the next task of machine in the same format of task , I will be happy to merge it.

Or I if you have other new ideas to add to the repo this will be awesome.

@AbdelrahmanKK